soc/renesas/ra/ra2/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 MUNIC SA
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_RA2
```

Comment:
It should be SOC_SERIES_RA2L1. See other PR for the naming convention.

Reply:
Completed. In the latest patch series.